robottelo/host_helpers/satellite_mixins.py

```diff
@@ -235,9 +235,9 @@ def available_capsule_port(self):
         :rtype: int
         """
         port_pool_range = settings.fake_capsules.port_range
-        if type(port_pool_range) is str:
+        if isinstance(port_pool_range, list):
```

Comment:
is it correct?
```suggestion
        if isinstance(port_pool_range, str):
```

Reply:
nice catch!

Reply:
@JacobCallahan Could you fix this in master branch manually?
I'll add similar comments to your other cherrypick PRs and we can commit this suggestion in same way

Reply:
manual fix required for master and 6.15.z branches*